src/views/profile/account-view/account-view.module.css

```diff
@@ -38,6 +38,4 @@
 
 .copySnippetWrapper {
 	width: max-content;
-	border-radius: 5px;
-	border: 1px solid var(--token-color-border-strong);
```

Comment:
remove double borders reported in [✓ Appears to be a double border on the "copy snippet" component on Profile info page](https://app.asana.com/0/0/1205523765248066/f)